invenio_app_ils/records/api.py

```diff
@@ -76,6 +76,17 @@ def create(cls, data, id_=None, **kwargs):
         return super(Item, cls).create(data, id_=id_, **kwargs)
 
 
+    def attach_document(self, document_pid, **kwargs):
+        """Attach a document ref to the record."""
+        self["document"] = {
+            "$ref": "{scheme}://{host}/api/resolver/items/document/<pid_value>".format(
```

Comment:
Maybe we should have this url in some variable, because it's repeated

Reply:
It's not fully repeated only the `api/resolver/` part, but creating only for this now I am not sure...Too many config variables after WDYT?